### ISSUE TITLE
scripts for OCR->BPTL level 1 conversion

### ISSUE THREAD

**Issue by emylonas** [kshawkin/Best-Practices-for-TEI-in-Libraries#88]
are we done with this? see #27

**Comment by sydb** [comment]
No, not even close. But then again, this is something I had not planned to finish before Tokyo. (Although I had planned to be further along than I am.) I do not think of these scripts as show-stoppers **at all**, because I think of them as ancillary materials. (Important, yes, but ancillary.)
